Add ads-twitter.com to disconnect_mapping.py

https://github.com/mozilla-services/shavar-prod-lists/pull/42 added ads-twitter.com under the `Disconnect` cateogry, but we didn't map it to a category in `disconnect_mapping.py`. This is causing `[ERROR] ads-twitter.com not found in disconnect_mapping` during `lists2safebrowsing.py`.

--- a/disconnect_mapping.py
+++ b/disconnect_mapping.py
@@ -25,6 +25,8 @@ disconnect_mapping = {
   "doubleclick.net": "Advertising",
   "adwords.google.com": "Advertising",
   "googleadservices.com": "Advertising",
+  "adservice.google.com": "Advertising",
+  "adservice.google.ca": "Advertising",
   "googlesyndication.com": "Advertising",
   "googletagservices.com": "Advertising",
   "googletagmanager.com": "Advertising",
@@ -56,5 +58,7 @@ disconnect_mapping = {
   # do we leave or remove?
 
   "crashlytics.com": "Analytics",
-  "tweetdeck.com": "Analytics"
+  "tweetdeck.com": "Analytics",
+
+  "ads-twitter.com": "Advertising",
 }

